app/models/crime_application.rb

```diff
@@ -11,10 +11,7 @@ def applicant_name
   def formatted_applicant_nino
     return if applicant.nino.nil?
 
-    # Remove all spaces
-    formatted_nino = applicant.nino.gsub(/\s+/, '').upcase
-    [2, 5, 8, 11].each { |i| formatted_nino.insert i, ' ' } if formatted_nino.length == 9
-    formatted_nino
+    applicant.nino
```

Comment:
Are we happy that Apply/the datastore removes any user-input spacing?

Reply:
Yep I checked the validation in Apply and it does